content/rs/getting-started/_index.md

```diff
@@ -74,10 +74,14 @@ To set up your machine as an RS cluster:
 
     ![Redis Enterprise Software create database](/images/rs/getstarted-newdatabase.png)
 
-1. Enter a database name such as `database1` and click **Activate** to create your database.
+1. Enter a database name such as `database1` and then select **Show Advanced Options**.
 
     ![Redis Enterprise Software configure new database screen](/images/rs/getstarted-createdatabase.png)
 
+1. In **Endpoint port number**, enter `12000`.
```

Comment:
Looks good, @lanceleonard what do you think about mentioning you can choose any port within the recommended range (10,000 - 19,999 as mentioned here https://docs.redislabs.com/latest/rs/administering/designing-production/networking/port-configurations/ )? e.g. 12000

Reply:
Let's talk about this as a future update.